### PR TITLE
Move spec to init

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**

### CODE REVIEW

**Comment by NicolasPetton** [comment]
Awesome! :trophy:  

Update the README as well maybe?

**Comment by BenjaminVanRyseghem** [comment]
done

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ var animal = object.subclass(function(that, spec, my) {
     };
 });
 
-var dog = animal.subclass(function(that, spec, my) {
+var dog = animal.subclass(function(thatf, my) {
```
Comment (NicolasPetton) [inline README.md]: typo

File: src/object.js [hunk 2]
```diff
@@ -118,8 +118,8 @@ define(function(){
 				extension(instance, spec, my);
```
Comment (NicolasPetton) [inline src/object.js]: we should remove spec from here too

**Pull request merged**